Turkey.Tests/NuGetTest.cs

```diff
@@ -58,19 +58,23 @@ public async Task LivePackagesAreIdentifiedCorrectly(string name, string version
         }
 
         [Theory]
-        [InlineData(new string[] { }, "")]
-        [InlineData(new string[] { "foo" }, "<add key=\"0\" value=\"foo\" />")]
-        [InlineData(new string[] { "foo" , "bar"}, "<add key=\"0\" value=\"foo\" /> <add key=\"1\" value=\"bar\" />")]
-        public void NuGetConfigIsGeneratedCorrectly(string[] urls, string feedParts)
+        [InlineData(new string[] { }, "", "3.1")]
+        [InlineData(new string[] { }, "", "5.0")]
+        [InlineData(new string[] { "foo" }, "<add key=\"0\" value=\"foo\" />", "3.1")]
+        [InlineData(new string[] { "foo" }, "<add key=\"0\" value=\"foo\" />", "5.0")]
+        [InlineData(new string[] { "foo" , "bar"}, "<add key=\"0\" value=\"foo\" /> <add key=\"1\" value=\"bar\" />", "3.1")]
+        [InlineData(new string[] { "foo" , "bar"}, "<add key=\"0\" value=\"foo\" /> <add key=\"1\" value=\"bar\" />", "5.0")]
+        public async Task NuGetConfigIsGeneratedCorrectly(string[] urls, string feedParts, string version)
         {
             using (var http = new HttpClient())
             {
                 NuGet nuget = new NuGet(http);
 
                 var expectedConfig = "<?xml version=\"1.0\" encoding=\"utf-8\"?> <configuration> <packageSources> " + feedParts + " </packageSources> </configuration>";
                 var cleanedExpectedConfig = Regex.Replace(expectedConfig, @"\s+", " ");
+                var nugetConfig = await new SourceBuild(http).GetNuGetConfigAsync(Version.Parse(version));
```

Comment:
This unit test is now dependent on the the network. Can somehow remove the dependency?

Reply:
mmmmhmmmmmmm

Reply:
Well I'd just remove the changes then. Faking the file makes no sense, it's already doing that.